assets/components/forms/formHOCs/withError.scss

```diff
@@ -1,9 +1,13 @@
 .component-error {
   display: block;
-  width: 324px;
+  width: calc(100vw - 36px);
```

Comment:
where does `36` come from?

Reply:
`20px` of margin + `16px` of padding.

Reply:
would using `100%` +`box-sizing: border-box` work for this? I'm a bit wary of replicating so many metrics this deep down

Reply:
You could apply `box-sizing: border-box` to avoid having to apply the reduction for padding, but I think it would still leave you with the margin to account for.

Reply:
what about `auto`? as a block this should take all the available space sans margins already right?